### PR TITLE
[KOGITO-1440] Add possibility to specify namespace name for Cucumber …

### PR DESCRIPTION
**Pull request by sutaakar**
…tests

https://issues.redhat.com/browse/KOGITO-1440

Signed-off-by: Karel Suta <ksuta@redhat.com>

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [x] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

File: test/framework/config.go [hunk 1]
```diff
@@ -172,3 +176,8 @@ func IsConfigDryRun() bool {
 func IsConfigKeepNamespace() bool {
 	return env.keepNamespace
 }
+
+// GetNamespaceName return namespace name if it was defined
+func GetNamespaceName() string {
```
Comment (radtriste) [inline test/framework/config.go]: all methods here are named `GetConfig[...]` or `IsConfig[...]`

File: test/steps/data.go [hunk 2]
```diff
@@ -46,12 +46,19 @@ func (data *Data) RegisterAllSteps(s *godog.Suite) {
...
 	data.StartTime = time.Now()
-	data.Namespace = framework.GenerateNamespaceName()
+	data.Namespace = getNamespaceName()
 
 	framework.GetLogger(data.Namespace).Info(fmt.Sprintf("Scenario %s", framework.GetScenarioName(s)))
 	go framework.StartPodLogCollector(data.Namespace)
 }
 
+func getNamespaceName() string {
+	if namespaceName := framework.GetNamespaceName(); namespaceName != "" {
```
Comment (radtriste) [inline test/steps/data.go]: please use `len(namespaceName) > 0`

**Comment by sutaakar** [comment]
@radtriste Updated

**Comment by sutaakar** [comment]
Right, https://github.com/kiegroup/kogito-cloud-operator/pull/210 should be merged first, then I need to adjust this PR.

**Comment by radtriste** [comment]
There will be conflict as well with https://github.com/kiegroup/kogito-cloud-operator/pull/208 ;)

**Comment by sutaakar** [comment]
Rebased to resolve conflicts.

**Pull request merged**